sdk/appconfiguration/app-configuration/src/internal/helpers.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ETagOption, ListConfigurationSettingsOptions, AppConfigurationGetKeyValuesOptionalParams } from '..';
+import { URLBuilder } from '@azure/core-http';
+import { isArray } from 'util';
+
+/**
+ * Formats the etag so it can be used with a if-match header
+ * @internal
+ * @ignore
+ */
+export function formatETagForMatchHeaders(objectWithEtag: ETagOption): string | undefined {
+  if (objectWithEtag.etag) {
+    return `"${objectWithEtag.etag}"`;
+  }
+
+  return undefined;
+}
+
+/**
+ * Transforms the keys/labels parameters in the listConfigurationSettings and listRevisions
+ * into the format the REST call will need.
+ * 
+ * @internal
+ * @ignore
+ */
+export function formatWildcards(
+  listConfigOptions?: ListConfigurationSettingsOptions
+): Pick<AppConfigurationGetKeyValuesOptionalParams, "key" | "label"> {
+  if (listConfigOptions == null) {
```

Comment:
we check for null, but null isn't a valid input (due to strictNulls) , only undefined. Also should be `===` if you wanted a real null check

Reply:
I was able to just remove it which was much nicer - I'd already changed all the callers to have a default value so this was unnecessary.